package.json

```diff
@@ -62,7 +62,7 @@
     "@emotion/eslint-plugin": "11.0.0",
     "@emotion/react": "11.1.4",
     "@emotion/styled": "11.0.0",
-    "@leafygreen-ui/badge": "4.0.5",
+    "@leafygreen-ui/badge": "7.0.0",
     "@leafygreen-ui/banner": "3.0.9",
```

Comment:
(placement of comment is random) I think it might make sense to also remove the excess padding on the badges that show under Failure Details. Here's the [patch on staging](http://localhost:3000/task/evg_coverage_coverage_patch_8f1a1622b544421afa3f29fb7d80376101bb28c8_5f6babf297b1d36af9f74b05_21_05_26_20_26_39/annotations?execution=0) I was looking at for this. I think the styling exists in two places, `BBComponents.tsx` and `AnnotationTicketRow.tsx`.

<img width="664" alt="Screen Shot 2022-10-31 at 3 00 00 PM" src="https://user-images.githubusercontent.com/47064971/199088474-24e5c92e-6bfa-47eb-b41e-c7f4da3ee132.png">

Reply:
Good catch! I think the `StyledBadge` margin in `AnnotationTicketRow.tsx` only applies spacing between badges but I'll update `BBComponents` 👍